Business Social Network/AppModules/Search/views/toolbar.html

```diff
@@ -58,19 +63,17 @@
 
       <form class="navbar-form navbar-default" role="search"
       ng-include src = "'partials/Search/searchBox.html'">
-
-      
-
     </form>
 
     <div id = "icons">
-      <ul class="nav navbar-nav navbar-right">
+      <ul class="nav navbar-nav navbar-right" ng-controller= 'WishListNotificationController'>
 
 
-        <li> <a href="#" class = "glyphicon glyphicon-shopping-cart"></a></li>
+        <li> <a href="#/wishlist/573f4399853cb2ca1affb6dc" class = "glyphicon glyphicon-shopping-cart"
```

Comment:
@HazemElAgaty 
user id should be passed dynamically or from current user session